### PR TITLE
look and say recursion algorithm integrated

### LINKED ISSUE #311: [Feature Request] Look and say problem solution in Recursion
### Is your feature request related to a problem? Please describe.
Currently recursion dont have famous look and say problem solution 

### Describe the solution you'd like
look and say is good problem to learn about recursion approach

### Additional context
image

---

**Would you like to work on this feature?**  
- [x] Yes  
- [ ] No

### PR DESCRIPTION
**Pull request by 17arindam**
### Description
Integrated look and say algorithm which contain Introduction Problem Statement Approach Pseudo Code Algorithm
Implementation in C++

Fixes #311 

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

### Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by 17arindam** [comment]
@ajay-dhangar please review my pr #311

File: docs/Recursion/look-and-say.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+int main() {
+    Solution obj;
+    int n = 5; // Let's find the 5th term in the Look and Say sequence
+    string result = obj.lookAndSay(n);
+    
+    // Output the result
+    cout << "The " << n << "th term of the Look and Say sequence is: " << result << endl;
+    return 0;
+}
```
Comment (ajay-dhangar) [inline docs/Recursion/look-and-say.md:94]: close your code example with ```` ``` ````

**Review by ajay-dhangar — CHANGES_REQUESTED** [review]
update your code based on suggestions

**Comment by 17arindam** [comment]
@ajay-dhangar i have corrected it please review

**Review by ajay-dhangar — APPROVED** [review]
Done

**Pull request merged**